src/Network/Network/help/Remove-AzNatGateway.md

```diff
@@ -38,8 +38,8 @@ Remove Nat Gateway Resource
 ### Example 1
 ```powershell
 PS C:> $nat = Get-AzNatGateway -ResourceGroupName "natgateway_test" -Name "nat_gateway"
-PS C:> Remove-AzNatGateway -InputObject $nat
```

Comment:
The cmdlets that have NatGateway as a noun should continue to have `-InputObject` and `-ResourceId`. Please revert the changes in examples and by just regenerating the help files the syntax should also be updated.

Reply:
Do this everywhere else applicable

Reply:
Updated this since we want to represent alias in the examples

Reply:
You can update the examples to use the alias instead of the original parameter name but because the syntax and parameter information are always replaced when the help files are regenerated we cannot update them manually.

In addition, the cmdlets that have NatGateway in the noun name should continue to follow ours guidelines using `-ResourceId` and `-InputObject`. Since the cmdlets have NatGateway in the noun it should be clear for users which ResourceId or InputObject these parameters refer to. However, as we talked before you can still add aliases for these parameters, nevertheless.

Reply:
Don't forget to regenerate all help files, please.

Reply:
@niander regenerated help files, please review.